src/common/cluster-ipc.ts

```diff
@@ -5,11 +5,10 @@ import { tracker } from "./tracker";
 export const clusterIpc = {
   activate: createIpcChannel({
     channel: "cluster:activate",
-    handle: (clusterId: ClusterId, frameId?: number) => {
+    handle: (clusterId: ClusterId, force = false) => {
```

Comment:
we don't activate cluster with `frameId` anymore, so it's safe to remove it from here and replace with `force`